src/sentry/templates/sentry/integrations/sentry-app-notify-disable.html

```diff
@@ -0,0 +1,23 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+  <p>
+    Hi there,
+</p><p>
+    We’ve received multiple <a href={{integration_link}}dashboard/>request errors</a> from your webhook: <a href={{webhook_url}}>{{webhook_url}}</a> for the custom {{ integration_name }} integration, and have unsubscribed the webhook from receiving events.
```

Comment:
We might want to add a referrer to the URL for analytics and as this is set up, it'll be http://dev.getsentry.net:8000/settings/sentry/developer-settings/internal-35e455/?referrer=notify_disable_emaildashboard/ (missing the `/`). We _could_ add the slash to the referrer query string but that seems odd, probably better to pass the dashboard URL to context.